src/types.ts

```diff
@@ -13,15 +14,15 @@ export type FallbackProps = {
   resetErrorBoundary: (...args: any[]) => void;
 };
 
-type ErrorBoundarySharedProps = {
+type ErrorBoundarySharedProps = PropsWithChildren<{
```

Comment:
After this change All ErrorBoundaryProps(ErrorBoundaryPropsWithFallback, ErrorBoundaryPropsWithComponent, ErrorBoundaryPropsWithRender) will contain children too.